spec/models/export/activity_change_state_column_spec.rb

```diff
@@ -0,0 +1,73 @@
+RSpec.describe Export::ActivityChangeStateColumn do
+  before(:all) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.start
+    @new_activity = create(:project_activity)
+    @changed_activity = create(:project_activity)
+    @unchanged_activity = create(:project_activity)
+  end
+
+  after(:all) do
+    DatabaseCleaner.clean
+  end
+
+  context "when there are rows" do
+    let(:report) { build_stubbed(:report) }
+    let(:activities) { [@new_activity, @changed_activity, @unchanged_activity] }
+
+    before do
+      changed_activities = double(ActiveRecord::Relation, pluck: [@changed_activity.id])
+      allow(report).to receive(:activities_updated).and_return(changed_activities)
+
+      new_activities = double(ActiveRecord::Relation, pluck: [@new_activity.id])
```

Comment:
As a general point I find it really satisfying when you can turn difficulty writing a simple test into a pointer to a better technical design. When it's hard to test it's usually because the design needs work. That's the second D in TDD!